examples/apps/spaces/src/fluid-object/index.tsx

```diff
@@ -100,27 +99,40 @@ export class Spaces extends DataObject implements IFluidHTMLView {
      * Will return a new Spaces View
      */
     public render(div: HTMLElement) {
-        if (this.storageComponent === undefined) {
-            throw new Error("Spaces can't render, storage not found");
-        }
-
         const addItem = (type: string) => {
             this.createAndStoreItem(type, { w: 20, h: 5, x: 0, y: 0 })
                 .catch((error) => {
                     console.error(`Error while creating item: ${type}`, error);
                 });
         };
 
+        const View: (props: any) => ReactElement = () => {
+            if (this.storageComponent === undefined) {
+                throw new Error("Spaces can't render, storage not found");
+            }
+            const [baseUrl, setBaseUrl] = React.useState<string | undefined>("");
+            React.useEffect(() => {
+                const getBaseUrl = async () => {
+                    setBaseUrl(await this.context.getAbsoluteUrl(this.handle.absolutePath));
+                };
+
+                void getBaseUrl();
```

Comment:
Is the usage of `void` here to avoid floating promise and/or uncaught promise exceptions?  Would be nice to catch the promise instead if so.

Reply:
ok, i can change that